### PR TITLE
Remove Sleeve System

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

A convoluted, sprite layering system that exists just to lets players tear the sleeves off only certain dresses. The real issue? Since every clothing sprite requires its sleeves to be separated in another icon file, It exponentially increases the TIME & COST of creating each new clothing piece. I don't think it's worth it at all.
Sleeve tearing is sacrificed for now but it can be reimplemented in a way thats much less aids.

- This additionally fixes all current missing-sleeves sprites.

### CODE REVIEW

**Comment by radarseas** [comment]
> why?

outfits render for the outfit itself, and each sleave as seperate textures. which is extrodinarily inefficient for making new ones and left several broken from a side-view where there was no sleaves rendering, its most noticable with noble-dresses or formal silks, or the vampire lord's regal silks without the cape.

**Comment by Coolgat3** [comment]
but why remove the quickest way to get cloth in an emergency situation.. i'd say leave it in just don't change sprite v.v

**Comment by DeerJesus** [comment]
leave the mechanic of sleeve tearing in just remove the different sprites

**Comment by TheFenix323** [comment]
> leave the mechanic of sleeve tearing in just remove the different sprites

The sleeve tearing system is still there

**Comment by Gyrandola** [comment]
I'm psychologically unable to bring this to completion for the time being, because I need to manually stitch back together all the sleeves to their corresponding main sprites.

**Comment by dwasint** [comment]
> > why?
> 
> outfits render for the outfit itself, and each sleave as seperate textures. which is extrodinarily **inefficient** for making new ones.
> 
> this left several broken from a side-view where there was no sleaves rendering, its most noticable with noble-dresses or formal silks, or the vampire lord's regal silks without the cape. where you can see the sleaves from the front but at a side-view, it doesn't cover your arms.
> 
> it also means for every new outfit you need to make the outfit _**AND**_ the _**SLEAVES**_ over the arms seperately.
> 
> TLDR: a small sacrifice for great progress

honestly if the reason is the workload I can make a script that splits icons into generic sleeves. or we just forgo that and use masks

**Comment by NPC1314** [comment]
Some sort of masking system would probably be the best solution, or just leave it as it is. While annoying it does actually work as designed.
